### PR TITLE
Added post about launchers

### PR DESCRIPTION
**Pull request by moochris**
I have written a short post about launchers. Hopefully it will look OK (is there a way I can preview this?)

### CODE REVIEW

**Comment by waddlesplash** [comment]
leafmenu.png and lnlauncher.png are HUGE files (~10MB ea) and they seem to be invalid ... ?

**Comment by waddlesplash** [comment]
Also, please put the images into the `static/files/` directory, and then link to them using `<a href="/files/..."`.

**Comment by moochris** [comment]
Really? Something must have gone wrong there - I'll take a look...

On 22 January 2017 17:37:36 GMT+00:00, waddlesplash <notifications@github.com> wrote:
>leafmenu.png and lnlauncher.png are HUGE files (~10MB ea) and they seem
>to be invalid ... ?
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/haiku/website/pull/20#issuecomment-274345508

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by waddlesplash** [comment]
Other than that, looks good to me. Maybe add a bit more of a conclusion than the abrupt ending you have now. I'd prefer someone else besides me review it though.

**Comment by pulkomandy** [comment]
Nothing about QuickLaunch? It would fit in there I think.

**Comment by moochris** [comment]
OK. The images were huge and not actually PNGs, because I saved over the top of them in Wonderbrush rather than using 'Export'... I can fix and put them under static/files/

I'll add something about QuickLaunch (forgot about that one) and end with a request for other suggestions in the forums (if I missed any others).

**Comment by waddlesplash** [comment]
s/forums/comments/, but yes, sounds good.

**Comment by moochris** [comment]
OK, all done. Sorry for the multiple commits - I need to get to grips with git a little more...

File: content/blog/moochris/2017-01-22_which_launcher.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+author = "moochris"
+date = "2017-01-22 12:05:49+00:00"
+tags = ["haiku", "software"]
++++
+
+<a href="/files/blog/moochris/leafmenu.png" title="Leaf Menu"><img src="/files/blog/moochris/leafmenu.png" title="Leaf Menu" alt="Leaf Menu" align="right"></a>So you've installed Haiku from a recently nightly (or sometime soon, the R1 beta) and you're launching applications from the 'leaf' menu. Perfect, but there are a few more options to investigate if you want to quickly launch your favourite programs.
+
+<!--more-->
+
+The first option is the simplest. In the 'leaf' application menu, just click on the 'Applications' submenu (or any of the others) to open a tracker window with shortcuts. Right-click on a shortcut and in the context menu select 'Create link', then 'Desktop'. Hey presto, a shortcut icon on your desktop.
```
Comment (humdingerb) [inline content/blog/moochris/2017-01-22_which_launcher.md]: Again, 'Deskbar menu' instead "'leaf' application menu". (Right-click-drag on Desktop is even easier, I think, but why not, for variety? :)

File: content/blog/moochris/2017-01-22_which_launcher.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+<a href="/files/blog/moochris/launchbox.png" title="LaunchBox"><img src="/files/blog/moochris/launchbox.png" title="LaunchBox" alt="LaunchBox" align="left"></a>This does require you to double-click the icon and the desktop can end up a bit messy this way :) Second option - use LaunchBox.
+
+LaunchBox can be found in 'Desktop applets' in the 'leaf' menu and you can make it start at boot by right-clicking on its icon and 'Create link' to /home/config/settings/boot/launch.
```
Comment (humdingerb) [inline content/blog/moochris/2017-01-22_which_launcher.md]: another "'leaf' menu"

File: content/blog/moochris/2017-01-22_which_launcher.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+<a href="/files/blog/moochris/quicklaunch.png" title="QuickLaunch"><img src="/files/blog/moochris/quicklaunch.png" title="QuickLaunch" alt="QuickLaunch" align="left"></a>Finally, you can always take a different approach and use QuickLaunch (again, available in the repository). With this tool, you simply start typing and you are presented with matching program names in a list. Hit enter to open the first result, or use the cursor keys to first select another. Simple and very handy, expecially if you use the 'Shortcuts' preferences to assign QuickLaunch to open on a key combination.
```
Comment (humdingerb) [inline content/blog/moochris/2017-01-22_which_launcher.md]: Not a flattering screenshot, but OK. :)

File: content/blog/moochris/2017-01-22_which_launcher.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
++++
+type = "blog"
+title = "Which Launcher?"
+author = "moochris"
+date = "2017-01-22 12:05:49+00:00"
+tags = ["haiku", "software"]
++++
+
+<a href="/files/blog/moochris/leafmenu.png" title="Leaf Menu"><img src="/files/blog/moochris/leafmenu.png" title="Leaf Menu" alt="Leaf Menu" align="right"></a>So you've installed Haiku from a recently nightly (or sometime soon, the R1 beta) and you're launching applications from the 'leaf' menu. Perfect, but there are a few more options to investigate if you want to quickly launch your favourite programs.
```
Comment (moochris) [inline content/blog/moochris/2017-01-22_which_launcher.md]: Thanks for reviewing - I'll make the changes to menu naming this evening.

File: content/blog/moochris/2017-01-22_which_launcher.md (hunk 1 above)
Comment (moochris) [inline content/blog/moochris/2017-01-22_which_launcher.md]: Hey, I didn't know you could do that! :)

File: content/blog/moochris/2017-01-22_which_launcher.md (hunk 3 above)
Comment (moochris) [inline content/blog/moochris/2017-01-22_which_launcher.md]: Feel free to add a nicer screenshot :)

File: content/blog/moochris/2017-01-22_which_launcher.md (hunk 1 above)
Comment (humdingerb) [inline content/blog/moochris/2017-01-22_which_launcher.md]: You can even create relative links by holding SHIFT while right-dragging. Tracker is really quite nice, but extending/changing stuff seems to be like unarming a bomb...

File: content/blog/moochris/2017-01-22_which_launcher.md (hunk 3 above)
Comment (humdingerb) [inline content/blog/moochris/2017-01-22_which_launcher.md]: Just a bit wider so the paths don't get truncated... doesn't really matter though.

File: content/blog/moochris/2017-01-22_which_launcher.md (hunk 4 above)
Comment (jessicah) [inline content/blog/moochris/2017-01-22_which_launcher.md]: Except it is a leaf. Using leaf here may help to reduce confusion when someone says to click on the leaf.

Comment (moochris) [inline content/blog/moochris/2017-01-22_which_launcher.md]: I could write, in the first instance: "Deskbar menu (the 'leaf' menu)"

Comment (humdingerb) [inline content/blog/moochris/2017-01-22_which_launcher.md]: or: "(the blue leaf of the Deskbar)". Then people click the blue _something_, even if they refuse to belief it's a leaf. :)

**Comment by humdingerb** [comment]
Thanks Chris. Do post again. :)

**Pull request merged**